src/firefly_exchange_client/client.py

```diff
@@ -789,6 +792,53 @@ async def get_user_leverage(self, symbol:MARKET_SYMBOLS, parentAddress:str=""):
         # todo fetch from exchange info route
         return 3
 
+    async def get_cancel_on_disconnect_timer(self,symbol:MARKET_SYMBOLS, parentAddress:str=""):
+        """
+            Returns a list of the user's countDowns for each market,
+            Inputs:
+                - symbol(MARKET_SYMBOLS): market symbol to get user market cancel_on_disconnect timer for.  
+                - parentAddress (str): Only provided by a sub account
+            Returns:
+                - GetCountDownsResponse:
+                    - countDowns: array with market symbols and countDown timer
```

Comment:
this will be array containing count down of all markets? we are sending a particular market symbol, shouldn't we be getting only that market's timer?